### DISCUSSION TITLE
Chromvar_clustering

### DISCUSSION THREAD

**Discussion by paulasoler** [stuart-lab/signac#698] [Q&A] [answered]
Hello @timoast !

I am trying to do a downstream analysis with the result provided by RunChromVAR. If I understand correctly, this function retrieves the deviation scores, the Z-score for each cell for each annotated motif. Then, using the following functions I have tried to know how the different cell types are related; so firstly, I have averaged the counts from chromVar for the cell identity and then I plotted (using pheatmap) the result of avgexpr_mat$chromvar.

avgexpr_mat <- AverageExpression(
  seurat,
  assays = "chromvar",
  return.seurat = F,
  group.by = "ident",
  slot = "data")

pheatmap(avgexpr_mat$chromvar, scale = "row")

As you could see from the heatmap, there is a clear biases towards two cell type where the majority of the signal is accumulated. So, can I do this type of analysis with the result of chromVar? Do I have to correct the data?
![Unknown](https://user-images.githubusercontent.com/3385292/124600353-e8481280-de66-11eb-87ff-2bb4c260db51.png)

Thanks for all

**Comment by timoast** [comment, marked as answer]
Rather than looking at every motif, I'd suggest finding differentially accessible motifs between  the cell types and then only plotting the top _n_ differential motifs per cell type.
